templates/css/product.css

```diff
@@ -49,8 +62,8 @@ amp-carousel {
 .header input[type=submit] {
   position:absolute;
   top:10px;
```

Comment:
Would it look OK to change this to space-1 or space-2?

Reply:
done